auth/auth.go

```diff
@@ -165,6 +166,10 @@ func (r *RenewingChecker) IsVPNExpired() (bool, error) {
 
 	if r.expChecker.IsExpired(data.ServiceExpiry) {
 		if err := r.fetchSaveServices(cfg.AutoConnectData.ID, &data); err != nil {
+			// TODO: now new account no vpn service is treated as if vpn service is expired
```

Comment:
Something is off with this comment, maybe a missing `with`?

```suggestion
			// TODO: from now on, new account with no vpn service is treated as if vpn service is expired
```